src/main/java/com/agilefaq/codedoctorsample/App.java

```diff
@@ -84,6 +84,12 @@ public static int multiply4(int a1, int a2) {
         if (a1 == 1) {
             a1 = 2;
         }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "a1". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1226)